heroku-cli/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM node:current-slim
 WORKDIR usr/src/app
 RUN npm install -g heroku
+RUN apt install python3 python3-pip
```

Comment:
we can pin the specific version of the heroku python package, as well as the npm package